src/routeHelpers.tsx

```diff
@@ -196,3 +196,21 @@ export type TypedParams = Record<string, string | undefined>;
 export const useTypedParams = <TParams extends TypedParams>() => {
   return useParams() as TParams;
 };
+
+export const myNdlaRoutes = {
+  myNdla: "/minndla",
+  myNdlaArena: "/minndla/arena",
+  myNdlaFolders: "/minndla/folders",
+  myNdlaSubjects: "/minndla/subjects",
+  myNdlaTags: "/minndla/tags",
+  myNdlaNotifications: "/minndla/arena/notifications",
+  myNdlaAdmin: "/minndla/admin",
+  myNdlaAdminFlags: "/minndla/admin/flags",
+  toMyNdlaArenaCategory: (categoryId: number) => `/minndla/arena/category/${categoryId}`,
+  toMyNdlaArenaTopic: (topicId?: number) => `/minndla/arena/topic/${topicId}`,
+  toMyNdlaArenaUser: (username: String) => `/minndla/arena/user/${username}`,
+  toMyNdlaFolder: (folderId: String) => `/minndla/folders/${folderId}`,
+  toMyNdlaFolderPreview: (folderId: String) => `/minndla/folders/preview/${folderId}`,
+  toMyNdlaAdminUsers: (params: String) => `/minndla/admin/users${params}`,
+  toMyNdlaAdminFlagsWithParams: (params: String) => `/minndla/admin/flags${params}`,
```

Comment:
Trenger vi `to`-prefixen på disse?

Reply:
nei, ikkje egentlig. Tenkte det berre var litt tydeligere, men kan endre.